omero/sysadmins/version-requirements.txt

```diff
@@ -2,193 +2,105 @@
     collections).
 
 * Python
 
-  * [5.2] 2.6 deprecated and not recommended, 2.7 recommended, 3.x not
-    supported
-  * [5.3] 2.6 dropped, 2.7 recommended, 3.x not supported
+  * [5.3] 2.6 dropped (OMERO.web only), 2.7 recommended, 3.x not supported
+  * [5.4] 2.6 dropped (OMERO.server), 2.7 recommended, 3.x not supported
   * Rationale: 2.7 is provided by all systems except for CentOS/RHEL
     6.x, however it is available officially via SCL. Note that 3.3 is
     also available via SCL, so there is potential for a 3.x migration
-    in the 5.3 timeframe. Dropping 2.6 for 5.3 unifies python support
-    with a single major version, and removes a number of portability
-    issues, but most importantly is a requirement of `Django 1.8`_.
-    Python 2.6 will continue to work for 5.2 with Django 1.6, but this
-    is not recommended due to security vulnerabilities.
-
-* GCC
-
-  * [5.2] 4.4 dropped, 4.6+ deprecated, 4.8+ recommended, 5 supported
-  * [5.3] 4.6+ dropped, 4.8 and 4.9 supported, 5 recommended
-
-  * Rationale: 4.4 is the CentOS/RHEL 6.x default, 4.8 the 7.x
-    default. The SCL DeveloperToolset3 brings CentOS 6 up to GCC
-    4.9.1, which makes the new baseline GCC 4.6 (Ubuntu 12.04 and
-    Travis). This will allow use of a limited subset of C++11 features
-    across all platforms in 5.2+. The downside is that while programs
-    and libraries built with the SCL DeveloperToolset3 will generally
-    be link-compatible with the system libraries, use of these
-    libraries with end user code will require the DeveloperToolset3
-    toolchain and libraries.
-
-The planned changes for 5.2 depend upon our support of
-    CentOS/RHEL 6.x and Ubuntu 12.04.
-
-* LLVM/clang
-
-  * [5.2] 3.4 and 3.5 supported, 3.6 recommended and 3.7 supported
-  * [5.3] 3.4 deprecated, 3.5 and 3.6 supported, 3.7 recommended
-  * Rationale: It is the current toolchain on MacOS/FreeBSD and is
-    working well; 3.3 dropped due to no longer being used by current
-    toolchains
-
-* Microsoft Visual Studio
-
-  * [5.2] 2010 dropped, 2012 deprecated, 2013 recommended,
-  * [5.3] 2012 deprecated, 2013 recommended, 2015 supported
-    2015 supported
-  * Rationale: The minimum (and maximum) MSVC version is determined by
-    the Windows Ice builds provided by ZeroC, and as such is limited
-    by our Ice version requirements. Raising the limit in 5.2 will
-    permit use of newer language features.
+    in the 5.3 timeframe. OMERO.web **no longer** works with Python 2.6,
+    the OMERO.server can still be used with Python 2.6.
 
 * Ice
 
-  * [5.2] 3.4 dropped, 3.5 recommended, 3.6 supported
-  * [5.3] 3.4 dropped, 3.5 deprecated, 3.6 recommended
-  * Rationale: 3.4 is no longer supported by ZeroC with 3.5 being
-    current. By the release of 5.3, Ice 3.6 will have been out for
-    some time and should be well tested.
+  * [5.3] 3.4 dropped, 3.5 deprecated, 3.6 recommended, 3.7 upcoming
+  * [5.4] 3.5 dropped, 3.6 supported, 3.7 recommended
```

Comment:
I will keep it as upcoming in that case since we have no idea either of the release timeline